client/components/title-format-editor/index.jsx

```diff
@@ -0,0 +1,307 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import {
+	map,
+	max,
+	min
+} from 'lodash';
+
+// The following polyfills exist for the draft-js editor, since
+// we are unable to change its codebase and yet we are waiting
+// on a solid solution for general IE polyfills
+//
+// They were borrowed and modified from MDN
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith
+if ( ! String.prototype.endsWith ) {
```

Comment:
In this case I think we're okay since all we care about is polyfilling the editor code, which loads via `require()` - do you think we run the risk here of breaking that assumption?

Actually I would love to nix all of this but I chose to keep it local here when I decided to put it in so that it will be more obvious where to remove once we have decided on proper polyfillery

thanks for the feedback @gwwar - I value your input